dlio_benchmark/utils/utility.py

```diff
@@ -51,12 +57,83 @@ def utcnow(format=LOG_TS_FORMAT):
     return datetime.now().strftime(format)
 
 
-def get_rank():
-    return MPI.COMM_WORLD.rank
-
-
-def get_size():
-    return MPI.COMM_WORLD.size
+# After the DLIOMPI singleton has been instantiated, the next call must be
+# either initialize() if in an MPI process, or set_parent_values() if in a
+# non-MPI pytorch read_threads child process.
+class DLIOMPI:
+    __instance = None
+    class MPI_state(Enum):
+        uninitialized = 1
```

Comment:
Also, check enumerations.py to move this there and call the enumerations

UNINITIALIZED = 0
MPI_INITIALIZED = 1 
CHILD_INITIALIZED = 2